PcAtChipsetPkg/PcatRealTimeClockRuntimeDxe/PcRtc.h

```diff
@@ -63,8 +63,24 @@ extern PC_RTC_MODULE_GLOBALS  mModuleGlobal;
 #define RTC_INIT_SECOND  0
 #define RTC_INIT_MINUTE  0
 #define RTC_INIT_HOUR    0
-#define RTC_INIT_DAY     1
-#define RTC_INIT_MONTH   1
+// MU_CHANGE [BEGIN] - Default date on the build date midnight.
+// Since we are in PST/PDT, if anyone between here and the international date line

```

Comment:
Is it legitimate to always assume we are in PST/PDT?

Reply:
https://learn.microsoft.com/en-us/cpp/preprocessor/predefined-macros?view=msvc-170

The __DATE__ uses the build system's time settings to get the date. The comment is just pointing out that the date only really applies for the set of time zones that fall into the day.

I think this is fine, it might give different results if its compiled in a data center in some other part of the world.

Do you want the comment changed?